### PR TITLE
fix: add field value to mixed case notice

### LINKED ISSUE #1392: Mixed case validation notice schema is incomplete
### Describe the problem

Mixed case validation was introduced in #1340.
In the validation report, we don't see the record values that trigger this notice, only the number of the row (see image below).
Seeing the actual value here would encourage the user to fix the issue because they would see what the rider would see.
Additionally, the `filename` name doesn't follow our naming conventions, it should be `fileName`.

Screenshot 2023-04-19 at 6 42 11 PM

### Proposed solution

I recommend adding a `FieldValue` in the schema that displays the value of the faulty record, and changing `filename` to `fileName` to follow our naming conventions.

 

### Alternatives you've considered

Status quo

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by briandonahue**
**Summary:**

Closes #1392

Add field value to notice.  Note, does not change casing of `filename` field as everywhere else is using the lowercase version.  

**Expected behavior:** 

Should show the offending value.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Run the unit tests with `gradle test` to make sure you didn't break anything
- [ ] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [ ] Linked all relevant issues
- [ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

**Comment by briandonahue** [comment]
Leaving this as a draft for now as it has some uncommitted changes that will be added in another PR, that will fix #1394

**Comment by briandonahue** [comment]
@isabelle-dr since that bug is resolved, this is ready for review. Let me know your thoughts on the `filename` casing as mentioned [here](https://github.com/MobilityData/gtfs-validator/issues/1392#issuecomment-1515555085)

**Comment by isabelle-dr** [comment]
Will have a look shortly, thank you. 
We'll also need a review from @davidgamez :)

File: web/service/build.gradle
Comment (davidgamez) [inline web/service/build.gradle:52]: question: do we need this change as part of the PR? It seems unrelated to the scope.

File: web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/Main.java
Comment (davidgamez) [inline web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/Main.java:39]: Same as the above comment. It looks unrelated to the scope of the PR.

**Comment by briandonahue** [comment]
@davidgamez I think this is fixed now. I think I had to pull in some of Kevin's changes to fix an issue I was having, but then afterward could not replicate the issue, so now I've removed the changes, and this should just be the original PR changes.

**Comment by davidgamez** [comment]
> @davidgamez I think this is fixed now. I think I had to pull in some of Kevin's changes to fix an issue I was having, but then afterward could not replicate the issue, so now I've removed the changes, and this should just be the original PR changes.

Great, thanks! Approving now!

**Review by davidgamez — APPROVED** [review]

**Pull request merged**
